src/apps/logging/js/src/logging.js

```diff
@@ -0,0 +1,58 @@
+function get_id_from_request_query(request) {
+  const elements = request.query.split("&");
+  for (const kv of elements) {
+    const [k, v] = kv.split("=");
+    if (k == "id") {
+      return ccf.strToBuf(v);
+    }
+  }
+  throw new Error("Could not find 'id' in query");
```

Comment:
Could we also add an example of logging to the node's log, just to showcase that feature?

Reply:
I don't think this JS app is (yet) a showcase for the full CCF-JS API. I think we could make it that (approach full parity with the C++ implementation, templated endpoints, different endpoint properties, etc), but for now I'd prefer to keep it as a very simple demo. The sample JS/TS apps do include `console.log` logging, so we show its possible and test it runs elsewhere.